### ISSUE TITLE
oewn:2024 doesn't download using verion 0.10.1 on PyPi

### ISSUE THREAD

**Issue by garymazz** [goodmami/wn#224]
**Describe the bug**
Downloading oewn:2024 fails with wn 0.10.1 on PyPi. I'm not sure if:

1. I need to locally build a new version
2. The is an issue with he documentation
3. I need to reference a different wn version on PyPI
4. The latest version needs push to PyPi

**To Reproduce**
$ python -m wn download oewn:2024
Traceback (most recent call last):
  File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/home/garym/.local/lib/python3.10/site-packages/wn/__main__.py", line 176, in <module>
    args.func(args)
  File "/home/garym/.local/lib/python3.10/site-packages/wn/__main__.py", line 18, in _download
    wn.download(target, add=args.add)
  File "/home/garym/.local/lib/python3.10/site-packages/wn/_download.py", line 59, in download
    cache_path, urls = _get_cache_path_and_urls(project_or_url)
  File "/home/garym/.local/lib/python3.10/site-packages/wn/_download.py", line 88, in _get_cache_path_and_urls
    info = config.get_project_info(project_or_url)
  File "/home/garym/.local/lib/python3.10/site-packages/wn/_config.py", line 165, in get_project_info
    raise ProjectError(f'no such version: {version!r} ({id})')
wn.ProjectError: no such version: '2024' (oewn)

**Expected behavior**
The command to download -oewn:2024 to complete successfully. 
eg:
Download [##############################] (12695101/12695101 bytes) Complete
Read [##############################] (1292452/1292452)
Added oewn:2024 (Open English WordNet)

**Environment**
python 3
Wn 0.10.1
omw-en31        1.4     [en]    OMW English Wordnet based on WordNet 3.1
oewn    2023    [en]    Open English WordNet

Additional Info:
$ python -m wn projects
i-      cili    1.0     [---]   Collaborative Interlingual Index
ic      oewn    2023    [en]    Open English WordNet
i-      oewn    2022    [en]    Open English WordNet
i-      oewn    2021    [en]    Open English WordNet
i-      ewn     2020    [en]    Open English WordNet
i-      ewn     2019    [en]    Open English WordNet
i-      odenet  1.4     [de]    Open German WordNet
i-      odenet  1.3     [de]    Open German WordNet
i-      omw     1.4     [mul]   Open Multilingual Wordnet
i-      omw-en  1.4     [en]    OMW English Wordnet based on WordNet 3.0
ic      omw-en31        1.4     [en]    OMW English Wordnet based on WordNet 3.1
i-      omw-arb 1.4     [arb]   Arabic WordNet (AWN v2)
i-      omw-bg  1.4     [bg]    BulTreeBank Wordnet (BTB-WN)
i-      omw-ca  1.4     [ca]    Multilingual Central Repository (Catalan)
i-      omw-cmn 1.4     [cmn-Hans]      Chinese Open Wordnet
i-      omw-da  1.4     [da]    DanNet
i-      omw-el  1.4     [el]    Greek Wordnet
i-      omw-es  1.4     [es]    Multilingual Central Repository (Spanish)
i-      omw-eu  1.4     [eu]    Multilingual Central Repository (Basque)
i-      omw-fi  1.4     [fi]    FinnWordNet
i-      omw-fr  1.4     [fr]    WOLF (Wordnet Libre du Français)
i-      omw-gl  1.4     [gl]    Multilingual Central Repository (Galician)
i-      omw-he  1.4     [he]    Hebrew Wordnet
i-      omw-hr  1.4     [hr]    Croatian Wordnet
i-      omw-id  1.4     [id]    Wordnet Bahasa (Indonesian)
i-      omw-is  1.4     [is]    IceWordNet
i-      omw-it  1.4     [it]    MultiWordNet (Italian)
i-      omw-iwn 1.4     [it]    ItalWordNet
i-      omw-ja  1.4     [ja]    Japanese Wordnet
i-      omw-lt  1.4     [lt]    Lithuanian  WordNet
i-      omw-nb  1.4     [nb]    Norwegian Wordnet (Bokmål)
i-      omw-nl  1.4     [nl]    Open Dutch WordNet
i-      omw-nn  1.4     [nn]    Norwegian Wordnet (Nynorsk)
i-      omw-pl  1.4     [pl]    plWordNet
i-      omw-pt  1.4     [pt]    OpenWN-PT
i-      omw-ro  1.4     [ro]    Romanian Wordnet
i-      omw-sk  1.4     [sk]    Slovak WordNet
i-      omw-sl  1.4     [sl]    sloWNet
i-      omw-sq  1.4     [sq]    Albanet
i-      omw-sv  1.4     [sv]    WordNet-SALDO
i-      omw-th  1.4     [th]    Thai Wordnet
i-      omw-zsm 1.4     [zsm]   Wordnet Bahasa (Malaysian)
i-      own     1.0.0   [mul]   Open Wordnets for Portuguese and English
i-      own-en  1.0.0   [en]    Open Wordnet for English
i-      own-pt  1.0.0   [pt]    Open Wordnet for Portuguese
i-      kurdnet 1.0     [ckb]   KurdNet (Kurdish WordNet)

**Comment by goodmami** [comment]
Thanks for the report. OEWN 2024 has been added to Wn's index in the `main` branch but I have not yet generated a release with it as I was trying to fix a few issues relevant to OEWN support first (see https://github.com/goodmami/wn/issues/221#issuecomment-2466075121). Unfortunately this means that you'll see OEWN 2024 on the README even though it's not yet released (note to myself: consider delaying README updates for the release commit).

Until I've made the release, you have two options:
1. Install Wn from the latest git repo version
2. Download OEWN:2024 by URL instead of project name

Of these, I think option (2) is better since the features I'm waiting on are a bit niche and you likely won't need them.

```bash
$ python -m wn download https://en-word.net/static/english-wordnet-2024.xml.gz
```
